src/otx/algo/common/losses/cross_focal_loss.py

```diff
@@ -79,7 +79,7 @@ def __init__(
 
         self.cls_criterion = cross_sigmoid_focal_loss
 
-    @custom_fwd(cast_inputs=torch.float32)
+    @custom_fwd(cast_inputs=torch.float32, device_type="cuda")
```

Comment:
Would this create an issue when we use XPU accelerator?

Reply:
It's going to be ignored, like it works now. The changes made do not change the actual behavior